package.json

```diff
@@ -49,7 +50,9 @@
     "rss-parser": "^3.9.0",
     "ts-jest": "^29.0.3",
     "type-graphql": "^1.1.1",
+    "typescript": "^4.0.5",
```

Comment:
why are we downgrading? document a reason in your pr description or revert this change